swagger.yaml

```diff
@@ -3786,6 +3786,72 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/clusters/{cluster_id}/ui-settings:
+    get:
+      tags:
+        - installer
+      description: Fetch cluster specific installer UI settings.
+      operationId: v2GetClusterInstallerUISettings
+      parameters:
+        - in: path
+          name: cluster_id
+          description: The cluster for which UI settings should be retrieved.
+          type: string
+          format: uuid
+          required: true
+      responses:
+        "200":
+          description: Success.
+          schema:
+            type: string
+        "400":
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
```

Comment:
In what case would 400 be returned?

Reply:
I think this is still a valid question. If you're not validating the input I'm not sure when we'd return 400.